README.md

```diff
@@ -34,7 +34,7 @@ After installing you can run:
 > http-api-docs
 ```
 
-This should spit out a Markdown document. This is exactly the api.md documentation at https://github.com/ipfs/website/blob/master/content/pages/docs/api.md , so you can redirect the output to just overwrite that file.
+This should spit out a Markdown document. This is exactly the api.md documentation at https://github.com/ipfs/website/blob/master/content/docs/api.md , so you can redirect the output to just overwrite that file.
```

Comment:
The correct URL for this is now 
https://docs.ipfs.io/reference/api/http/

Reply:
I think it's helpful to link to the actual output markdown file (perhaps in addition to the new human readable url you pasted) - which is something I spent some time looking for.  I'm looking at converting this lib into something that outputs a Python file as a part of a Python 3 asyncio library - and being able to that resulting output file was helpful FWIW

Reply:
Oh, in that case, you want 
https://github.com/ipfs/docs/blob/master/content/reference/api/http.md instead. All the docs inside the website repo are deprecated at this point.

Reply:
Ahh - great, thanks!  

Guess this PR is not so useful now - feel free to just close it and update the README w/ whatever fixed URL works best.  Thanks!